cernopendata/templates/cernopendata_theme/footer.html

```diff
@@ -4,65 +4,65 @@
       <div class="six wide column">
         <div class="ui list horizontal logos">
           <div class="item">
-            <a href="http://alice.cern"
-              ><img
+            <a href="http://alice.cern" aria-label="ALICE"
+            ><img
                 src="{{ url_for('static', filename='img/alice.gif') }}"
-                alt=""
+                alt="ALICE experiment logo"
```

Comment:
I know that having an alt text like `"Image of ..."` or `"Picture of ..."` is a bad practice, because the screen reader will tell the user that it is an image.
On the other hand, I do not know if `"ALICE experiment"` is better or not than `"ALICE experiment logo"`.

Reply:
Done, thanks!